index.js

```diff
@@ -13,9 +14,19 @@ const classnames = props => Object.keys(props)
   .filter(key => props[key] === true)
   .join(' ')
 
-class Base extends React.Component {
+class Base extends Component {
+  static propTypes = {
+    className: PropTypes.string,
```

Comment:
Thanks, but this is an HTML attribute, so React should handle this without a prop type definition

Reply:
it's true :|